codec_specs.md

```diff
@@ -642,6 +644,14 @@ Description: DTS Lossless audio that does not have a core substream. The private
 
 Initialization: none
 
+### A_EAC3
+
+Codec ID: A_EAC3
+
+Codec Name: Dolby Digital Enhanced AC-3
```

Comment:
```suggestion
Codec Name: Dolby Digital / DD+ / Enhanced AC-3 / E-AC-3
```

"Dolby Digital Enhanced AC-3" has no sense as "Dolby Digital" references AC-3 only. See e.g. https://professional.dolby.com/technologies/dolby-digital-plus/, "Professional tab, it is explicit: "The Dolby Digital Plus™ codec (E-AC-3)", also the comparison tab indicates "Dolby Digital" as a description of AC-3.

Reply:
As stated above, should be `Dolby Digital Plus (E-AC-3)` in my opinion.